examples/bomex.jl

```diff
@@ -0,0 +1,405 @@
+
+θ = liquid_ice_potential_temperature(model)
+u_avg = Field(Average(model.velocities.u, dims=(1, 2)), data=u_avg.data)
+v_avg = Field(Average(model.velocities.v, dims=(1, 2)), data=v_avg.data)
+θ_avg = Field(Average(θ, dims=(1, 2)), data=θ_avg.data)
+qᵗ_avg = Field(Average(model.specific_moisture, dims=(1, 2)), data=qᵗ_avg.data)
+
+function compute_averages!(sim)
+    compute!(u_avg)
+    compute!(v_avg)
+    compute!(θ_avg)
+    compute!(qᵗ_avg)
+    return nothing
+end
+
+add_callback!(simulation, compute_averages!)
+
+# ## Output and progress
+#
+# We output horizontally-averaged profiles for post-processing.
+
+qˡ = model.microphysical_fields.qˡ
+qᵛ = model.microphysical_fields.qᵛ
+
+function progress(sim)
+    qˡmax = maximum(qˡ)
+    qᵗmax = maximum(sim.model.specific_moisture)
+    umax = maximum(abs, u_avg)
+    vmax = maximum(abs, v_avg)
+
+    msg = @sprintf("Iter: %d, t: %s, Δt: %s, max|ū|: (%.2e, %.2e), max(qᵗ): %.2e, max(qˡ): %.2e",
+                   iteration(sim), prettytime(sim), prettytime(sim.Δt),
+                   umax, vmax, qᵗmax, qˡmax)
+    @info msg
+    return nothing
+end
+
+add_callback!(simulation, progress, IterationInterval(100))
+
+outputs = merge(model.velocities, model.tracers, (; θ, qˡ, qᵛ))
+averaged_outputs = NamedTuple(name => Average(outputs[name], dims=(1, 2)) for name in keys(outputs))
+
+filename = "bomex.jld2"
+simulation.output_writers[:averages] = JLD2Writer(model, averaged_outputs; filename,
+                                                  schedule = AveragedTimeInterval(20minutes, window=20minutes, stride=1),
+                                                  overwrite_existing = true)
+
+@info "Running BOMEX simulation..."
+run!(simulation)
+
+# ## Results: mean profile evolution
+#
+# We visualize the evolution of horizontally-averaged profiles every 20 minutes, similar
+# to Figure 3 in the paper by [Siebesma2003](@cite). The intercomparison study shows
+# that after spin-up, the boundary layer reaches a quasi-steady state with:
+# - A well-mixed layer below cloud base (~500 m)
+# - A conditionally unstable cloud layer (~500-1500 m)
+# - A stable inversion layer (~1500-2000 m)
+
+θt = FieldTimeSeries(filename, "θ")
+qᵛt = FieldTimeSeries(filename, "qᵛ")
+qˡt = FieldTimeSeries(filename, "qˡ")
+ut = FieldTimeSeries(filename, "u")
+vt = FieldTimeSeries(filename, "v")
+
+for fts in (qᵛt, qˡt)
+    fts .*= 1000 # convert kg/kg -> g/kg
+end
+
+times = θt.times
+Nt = length(times)
+
+# Create a 2×2 panel plot showing the evolution of key variables
+
+fig = Figure(size=(900, 800), fontsize=14)
+
+axθ = Axis(fig[1, 1], xlabel="θ (K)", ylabel="z (m)")
+axq = Axis(fig[1, 2], xlabel="qᵛ (g/kg)", ylabel="z (m)")
+axuv = Axis(fig[2, 1], xlabel="u, v (m/s)", ylabel="z (m)")
+axqˡ = Axis(fig[2, 2], xlabel="qˡ (g/kg)", ylabel="z (m)")
+
+for n in 1:Nt
+    t_max = Int(times[n] / 60)
+
+    label = n == 1 ? "t = $t_max min" : "mean $(t_max - 20)-$t_max min"
```

Comment:
I'm confused about this label, what is it trying to say?

Reply:
[39172a1](https://github.com/NumericalEarth/Breeze.jl/pull/228/commits/39172a1e365fc83399258636907e248c8a8f86d0) ?

Reply:
yes i like that better!